src/general/base_stage/CreatureStageHUDBase.cs

```diff
@@ -1088,6 +1105,8 @@ protected override void Dispose(bool disposing)
             }
 
             fadeParameterName.Dispose();
+
+            redFill?.Dispose();
```

Comment:
This line should be within the `if (disposing)` statement.

Also I'm a tiny bit suspicious if this is required, but I suppose it is because when the stylebox is not attached as the theme override, this is an unatached resource.

Reply:
> This line should be within the if (disposing) statement.

I think it already is

Reply:
Ah right it is, now that I'm looking again I did indeed misread the number of close braces.